src/plugins/intel_gpu/src/graph/gemm.cpp

```diff
@@ -186,7 +227,13 @@ std::vector<layout> gemm_inst::transform_input_layouts(const std::shared_ptr<con
 
     std::vector<layout> layouts = input_layouts;
     layouts[0].set_partial_shape(updated_input0_pshape);
+    if (primitive->input0_target_shape.size() > input_rank) {
+        layouts[0].format = format::adjust_to_rank(layouts[0].format, input_rank);
+    }
     layouts[1].set_partial_shape(updated_input1_pshape);
+    if (primitive->input1_target_shape.size() > weight_rank) {
+        layouts[1].format = format::adjust_to_rank(layouts[1].format, weight_rank);
+    }
```

Comment:
Seems that 
- get_updated_input_shape : for transpose 
- get_fused_op.. : braodcast 

However the namings are not quite intuitive.. how do you think? 
I would like to ask to rename functions and variables properly so that we could quickly catch each purpose.

Reply:
Updated it to be more intuitive